### PR TITLE
Dependencies and some styling

### PR DESCRIPTION
**Pull request by ohadbitt**
**Fixes:**
- Exclude conflicting dependencies

### CODE REVIEW

File: connector/src/main/scala/com/microsoft/kusto/spark/utils/DataTypeMapping.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ object DataTypeMapping {
     "object" -> StringType
   )
 
-  val sparkTypeToKustoTypeMap: Map[DataType, String] = Map(
+  val SparkTypeToKustoTypeMap: Map[DataType, String] = Map(
```
Comment (yihezkel) [inline connector/src/main/scala/com/microsoft/kusto/spark/utils/DataTypeMapping.scala:35]: This seems to duplicate KustoTypeToSparkTypeMap
THis is out of scope for your change, but just a note that it seems we can use Guava's BiMap here to remove the duplication.

File: connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoConstants.scala [hunk 2]
```diff
@@ -4,18 +4,18 @@ import scala.concurrent.duration._
...
-  val oneMega: Int = oneKilo * 1024
-  val oneGiga: Int = oneMega * 1024
+  val DefaultWaitingIntervalLongRunning: String = (2 days).toSeconds.toString
+  val DefaultPeriodicSamplePeriod: FiniteDuration = 1 seconds
+  val DefaultIngestionTaskTime: FiniteDuration = 20 seconds
+  val ClientName: String = KustoDataSourceUtils.clientName
+  val DefaultBufferSize: Int = 16 * 1024
+  val StorageExpiryMinutes: Int = 120
+  val SparkSettingsRefreshMinutes: Int = 120
+  val OneKilo: Int = 1024
```
Comment (yihezkel) [inline connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoConstants.scala:14]: nit: units are left out. Recommend including the unit (e.g. OneKilobit)

**Review by yihezkel — APPROVED** [review]

File: connector/src/main/scala/com/microsoft/kusto/spark/utils/DataTypeMapping.scala (hunk 1 above)
Comment (ohadbitt) [reply connector/src/main/scala/com/microsoft/kusto/spark/utils/DataTypeMapping.scala:35]: its not 1 to 1

File: connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoConstants.scala (hunk 2 above)
Comment (ohadbitt) [reply connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoConstants.scala:14]: OneKilobite ...

**Pull request merged**